### DISCUSSION TITLE
Get payment_type_id from onSubmit or getFormData()

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-react#40] [Q&A] [answered]
Hi, when I trigger `onSubmit` callback or `getFormData` it returns an object containing useful information to send to backend, I notice that the `payment_type_id` (credit_card, debit_card) is not available in this object.  
There is a way to obtain the `payment_type_id`, since this could be a useful information to send some metrics on my app.

Thanks 😃

**Comment by meliguicarvalho** [comment, marked as answer]
Hello!
I assume you are referring to the `Card Payment Brick`. We currently do not have a way to retrieve the `payment_type_id ` from this Brick.

However, this is an interesting suggestion, we will analise the possibility of adding this data, and report back to you in this thread.

If you need this data for your integration, I can suggest a workaround that will allow you to obtain it, if and while we do not provide this information in a Brick callback.

You could call the `payment_methods` [API](https://api.mercadopago.com/v1/payment_methods/search?public_key=<YOUR_PUBLIC_KEY>) directly and obtain the list of payment methods. There you can check the payment method's `id`  and the corresponding `payment_type_id`. With this data you can get the `payment_type_id` based on the `payment_method_id` you obtain from the Brick's callback.
